llmfoundry/callbacks/curriculum_learning_callback.py

```diff
@@ -28,9 +28,9 @@ class CurriculumLearning(CallbackWithConfig):
     """Starts an epoch with a different dataset when resuming from a checkpoint.
 
     Args:
-        dataset_index (int): The index of the dataset currently being used.
-        current_dataset_config (Dict): The configuration of the dataset currently
+        train_config (Dict): The configuration of the dataset currently
```

Comment:
Note that this is the full train config and must contain a `train_loader` key

Reply:
do you want me to note that in the docstring? or just noting here

Reply:
in the docstring plz

Reply:
done